test/EDSC/builder-api-test.cpp

```diff
@@ -746,6 +747,41 @@ TEST_FUNC(empty_map_load_store) {
   f.erase();
 }
 
+// CHECK-LABEL: func @affine_if_op
+// CHECK:       affine.if ([[d0:.*]], [[d1:.*]]){{\[}}[[s0:.*]], [[s1:.*]]{{\]}}
+// CHECK-NEXT:  }
+// CHECK:       affine.if ([[d0:.*]], [[d1:.*]]){{\[}}[[s0:.*]], [[s1:.*]]{{\]}}
+// CHECK-NEXT:  } else {
+TEST_FUNC(affine_if_op) {
+  using namespace edsc;
+  using namespace edsc::intrinsics;
+  using namespace edsc::op;
+  auto f32Type = FloatType::getF32(&globalContext());
+  auto memrefType = MemRefType::get({-1, -1}, f32Type, {}, 0);
+  auto f = makeFunction("affine_if_op", {}, {memrefType});
+
+  OpBuilder builder(f.getBody());
+  ScopedContext scope(builder, f.getLoc());
+  // clang-format off
```

Comment:
Is this intentional? Can you add a comment on why?
It is usually done exceptionally for well scoped block of code, but here it isn't clear to me that the following declaration needs it and I don't see the end of the unformatted region.

Reply:
I removed the clang-format. Are you also referring to ScopedContext ? Without it the EDSC builders crash.

Reply:
No I was just referring to the clang-format directive. Thanks.